### DISCUSSION TITLE
AkienFX Youtube Video

### DISCUSSION THREAD

**Discussion by JazzyJazzjazzy** [T-Troll/alienfx-tools#283] [General]
I notice there are no videos on Youtube showing how to use AlienFx. Any plans?

**Comment by T-Troll** [comment]
Being honest, i hate video guides - it's just a waste of time, read some text much faster.
So no, no plans for it. You can make some if you wish.

**Reply by JazzyJazzjazzy** [reply]
Honestly? I would but I have had little success using AlienFx. I have read the documentation and I still am uncertain if my device is recognized. I wanted to watch for my own benefit. I use it to turn my lights off and on. My device is an Alienware m17 R4.

**Reply by T-Troll** [reply]
> am uncertain if my device is recognized

From the [Beginner's guide](https://github.com/T-Troll/alienfx-tools/wiki/How-to-start-%28Beginner%27s-guide%29) - "You will see hardware devices present into the system at the left-top list. But for the first start, no lights defined yet, so you can press "Detect Devices" button if you want to look up your gear into my database."

17R4 is into database... Even don't need manual setup.
